test/Recipe-test.js

```diff
@@ -0,0 +1,56 @@
+import { expect } from 'chai';
+import Recipe from '../src/classes/Recipe';
+import { sampleIngredientsData, sampleUsersData, sampleRecipeData } from '../src/data/sample-data';
+
+describe('Recipe', () => {
+  sampleIngredientsData
+  sampleUsersData
+  sampleRecipeData
+  let recipe
+
+
+  beforeEach(() => {
+    recipe = new Recipe(sampleRecipeData[0]);
+    sampleIngredientsData;
+    sampleUsersData;
+    sampleRecipeData;
+  });
+
+  it('Should be a function', () => {
+    expect(Recipe).to.be.a('function');
+  });
+  it('should hold property of id', () => {
+    expect(recipe.id).to.equal(sampleRecipeData[0].id)
```

Comment:
hardcode to.equals to a visible data value. Example: expect(recipe.id).to.equal(20081). Repeat this for lines 26, 29, 32,  35, and 38.